index.html

```diff
@@ -35,10 +37,22 @@
   </header>
 
     <main>
-
+      <div class="counter">
+        <button class="counter__sign" id="counter__sign--minus">
```

Comment:
no need to use BEM convention in `ids` :) `camelCase`, `kebab-case` or `snake_case` are most popular and good practices here 👍